pkgs/development/libraries/rapidfuzz-cpp/default.nix

```
@@ -33,6 +34,13 @@ stdenv.mkDerivation (finalAttrs: {
     catch2_3
   ];
 
+  passthru = {
+    tests = {
+      /** `python3Packages.levenshtein` crucially depends on `rapidfuzz-cpp` */
+      levenshtein = python3Packages.levenshtein;
```

Can be shortened as
```suggestion
      inherit (python3Packages) levenshtein;
```
if you feel like it.